docker/docker-compose.yml

```diff
@@ -93,6 +94,12 @@ services:
         hard: -1
     volumes:
       - data01:/usr/share/opensearch/data
+#    To ensure opensearch is ready for localstack-init commands
```

Comment:
I was having issues when freshly bringing up the app when opensearch wasn't ready so I've added this to make sure it's ready for the indices to be created.

Reply:
you can probably remove the health check from localstack then, that will be left over from when we were using the built-in opensearch